apps/provider-console/src/components/ui/dropdown-menu.tsx

```diff
@@ -0,0 +1,195 @@
+"use client";
```

Comment:
Please use the components in the ui package :)

Reply:
Yes, this is planned in the next rebase.

Reply:
Ok great